src/Window.h

```diff
@@ -180,11 +181,11 @@ class Window : public QMainWindow {
     int mazeWidth();
     int mazeHeight();
 
-    bool wallFront();
+    bool wallFront(bool useDistance = false);
```

Comment:
Can you change this to `int distance`? My preference is to avoid default values for arguments, which means you'll need to explicit pass `1` when the user calls `wallFront`.